[2014-07-06 19:04:56] log.ERROR: exception 'PDOException' with message 'SQLSTATE[HY000] [2002] No connection could be made because the target machine actively refused it. ' in D:\zend\Apache2\htdocs\laravel.com - Copy\vendor\laravel\framework\src\Illuminate\Database\Connectors\Connector.php:47 Stack trace: #0 D:\zend\Apache2\htdocs\laravel.com - Copy\vendor\laravel\framework\src\Illuminate\Database\Connectors\Connector.php(47): PDO->__construct('mysql:host=loca...', 'root', 'admin', Array) #1 D:\zend\Apache2\htdocs\laravel.com - Copy\vendor\laravel\framework\src\Illuminate\Database\Connectors\MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array) #2 D:\zend\Apache2\htdocs\laravel.com - Copy\bootstrap\compiled.php(6644): Illuminate\Database\Connectors\MySqlConnector->connect(Array) #3 D:\zend\Apache2\htdocs\laravel.com - Copy\bootstrap\compiled.php(6571): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql') #4 D:\zend\Apache2\htdocs\laravel.com - Copy\bootstrap\compiled.php(6545): Illuminate\Database\DatabaseManager->makeConnection('mysql') #5 D:\zend\Apache2\htdocs\laravel.com - Copy\bootstrap\compiled.php(6433): Illuminate\Database\DatabaseManager->connection(NULL) #6 D:\zend\Apache2\htdocs\laravel.com - Copy\bootstrap\compiled.php(6420): Illuminate\Database\Eloquent\Model::resolveConnection(NULL) #7 D:\zend\Apache2\htdocs\laravel.com - Copy\bootstrap\compiled.php(6058): Illuminate\Database\Eloquent\Model->getConnection() #8 D:\zend\Apache2\htdocs\laravel.com - Copy\bootstrap\compiled.php(6031): Illuminate\Database\Eloquent\Model->newBaseQueryBuilder() #9 D:\zend\Apache2\htdocs\laravel.com - Copy\vendor\laravel\framework\src\Illuminate\Auth\EloquentUserProvider.php(87): Illuminate\Database\Eloquent\Model->newQuery() #10 D:\zend\Apache2\htdocs\laravel.com - Copy\vendor\laravel\framework\src\Illuminate\Auth\Guard.php(300): Illuminate\Auth\EloquentUserProvider->retrieveByCredentials(Array) #11 [internal function]: Illuminate\Auth\Guard->attempt(Array) #12 D:\zend\Apache2\htdocs\laravel.com - Copy\bootstrap\compiled.php(6924): call_user_func_array(Array, Array) #13 D:\zend\Apache2\htdocs\laravel.com - Copy\bootstrap\compiled.php(3474): Illuminate\Support\Manager->__call('attempt', Array) #14 D:\zend\Apache2\htdocs\laravel.com - Copy\bootstrap\compiled.php(3474): Illuminate\Auth\AuthManager->attempt(Array) #15 D:\zend\Apache2\htdocs\laravel.com - Copy\app\controllers\LoginController.php(45): Illuminate\Support\Facades\Facade::__callStatic('attempt', Array) #16 D:\zend\Apache2\htdocs\laravel.com - Copy\app\controllers\LoginController.php(45): Illuminate\Support\Facades\Auth::attempt(Array) #17 [internal function]: App\Controllers\LoginController->ProsesLogin() #18 D:\zend\Apache2\htdocs\laravel.com - Copy\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(138): call_user_func_array(Array, Array) #19 D:\zend\Apache2\htdocs\laravel.com - Copy\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('ProsesLogin', Array) #20 D:\zend\Apache2\htdocs\laravel.com - Copy\bootstrap\compiled.php(4952): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'ProsesLogin', Array) #21 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #22 D:\zend\Apache2\htdocs\laravel.com - Copy\bootstrap\compiled.php(8178): call_user_func_array(Object(Closure), Array) #23 D:\zend\Apache2\htdocs\laravel.com - Copy\bootstrap\compiled.php(8165): Illuminate\Routing\Route->callCallable() #24 D:\zend\Apache2\htdocs\laravel.com - Copy\bootstrap\compiled.php(4963): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #25 D:\zend\Apache2\htdocs\laravel.com - Copy\bootstrap\compiled.php(534): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #26 D:\zend\Apache2\htdocs\laravel.com - Copy\bootstrap\compiled.php(521): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #27 D:\zend\Apache2\htdocs\laravel.com - Copy\public\index.php(49): Illuminate\Foundation\Application->run() #28 D:\zend\Apache2\htdocs\laravel.com - Copy\server.php(19): require_once('D:\zend\Apache2...') #29 {main} [] []
[2014-07-06 19:05:38] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException' in D:\zend\Apache2\htdocs\laravel.com - Copy\bootstrap\compiled.php:5132 Stack trace: #0 D:\zend\Apache2\htdocs\laravel.com - Copy\bootstrap\compiled.php(4974): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\MethodNotAllowedException)) #1 D:\zend\Apache2\htdocs\laravel.com - Copy\bootstrap\compiled.php(4962): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 D:\zend\Apache2\htdocs\laravel.com - Copy\bootstrap\compiled.php(534): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 D:\zend\Apache2\htdocs\laravel.com - Copy\bootstrap\compiled.php(521): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 D:\zend\Apache2\htdocs\laravel.com - Copy\public\index.php(49): Illuminate\Foundation\Application->run() #5 D:\zend\Apache2\htdocs\laravel.com - Copy\server.php(19): require_once('D:\zend\Apache2...') #6 {main} [] []
